nomad/conf/validators.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
You're right. This is just that old habit of writing Python 2/3 compatible code.